pkg/istiovalues/profiles.go

```diff
@@ -52,6 +52,21 @@ func ApplyUserValues(mergedValues, userValues helm.Values,
 	return values, nil
 }
 
+// ApplyFipsValues: Detect FIPS enabled mode on OpenShift and add pilot.env.COMPLIANCE_POLICY
+func ApplyFipsValues(mergedHelmValues helm.Values, checkFilePath string) (helm.Values, error) {
```

Comment:
This should probably go into `fips.go` instead of `profiles.go`, since it has nothing to do with profiles.

Also, why does the filepath need to be passed in? It's a well-known path, so it could be defined in a constant in `fips.go`.

Reply:
@luksa I updated my PR about the two suggestions above. Thanks, Marko.